content/docs/user-guide/basic-concepts/parameter.md

```diff
@@ -5,5 +5,5 @@ tooltip: >-
   Pipeline stages (defined in `dvc.yaml`) can depend on specific values inside
   an arbitrary YAML, JSON, TOML, or Python file (`params.yaml` by default).
   Stages are invalid (considered outdated) when any of their parameter values
-  change. See `dvc params`.
+  change. See [`dvc params`](/doc/command-reference/params).
```

Comment:
These get linked automatically, no need for md link (I removed it again). Please sanity test changes in a local dev env. Thanks

Reply:
No they're not (see other basic-concepts content too). Maybe for separate issue to fix that.

Reply:
Haha you got me m8. I was assuming this wasn't visually checked since the PR isn't deploying. My bad!